File: algorithms/Codility-master/Codility-master/9_MaximumSliceProblem/MaxDoubleSliceSum.py
# 100%
def solution(A):
    # write your code in Python 3.6
    L_slice, R_slice, count = 0, 0, 0

    for Z in range(3, len(A)):
        L_slice = max(0, A[Z-2] + L_slice)
        R_slice = max(L_slice, A[Z-1] + R_slice)
        count = max(R_slice, count)

    return count

File: algorithms/Codility-master/Codility-master/9_MaximumSliceProblem/MaxDoubleSliceSum.js
// 100%
function solution(A) {
    // write your code in JavaScript (Node.js 8.9.4)
    let L_slice = 0
    let R_slice = 0;
    let count = 0;
    for (let Z=3; Z < A.length; Z++){
        L_slice = Math.max(0, A[Z-2] + L_slice);
        R_slice = Math.max(L_slice, A[Z-1] + R_slice);
        count = Math.max(R_slice, count);
    }
     return count;
}